homeassistant/components/vacuum/mqtt.py

```diff
@@ -92,35 +93,21 @@ def strings_to_services(strings):
 DEFAULT_NAME = 'MQTT Vacuum'
 DEFAULT_RETAIN = False
 DEFAULT_SERVICE_STRINGS = services_to_strings(DEFAULT_SERVICES)
-DEFAULT_COMMAND_TOPIC = 'vacuum/command'
 DEFAULT_PAYLOAD_TURN_ON = 'turn_on'
 DEFAULT_PAYLOAD_TURN_OFF = 'turn_off'
 DEFAULT_PAYLOAD_RETURN_TO_BASE = 'return_to_base'
 DEFAULT_PAYLOAD_STOP = 'stop'
 DEFAULT_PAYLOAD_CLEAN_SPOT = 'clean_spot'
 DEFAULT_PAYLOAD_LOCATE = 'locate'
 DEFAULT_PAYLOAD_START_PAUSE = 'start_pause'
-DEFAULT_BATTERY_LEVEL_TOPIC = 'vacuum/state'
```

Comment:
Removing sane defaults is a bummer to me. Every home assistant user that wants to use this component will now need to add an at least ~10 line config block to `configuration.yaml`. I bet this configuration will be exactly the same (copied and pasted from the github.io docs example) for 80% of people.

Reply:
@MartinHjelmare what do you think?